features/step_definitions/view_dishes_steps.rb

```diff
@@ -0,0 +1,14 @@
+
+Given(/^there is a dish called "([^"]*)" on "([^"]*)"$/) do |dishName, menuName|
```

Comment:
This is nitpicking, but in Ruby we write variables like `dish_name` -> snake case